src/OrbitGl/App.h

```diff
@@ -289,11 +289,20 @@ class OrbitApp final : public DataViewFactory, public CaptureListener {
       const std::vector<ModuleData*>& modules,
       absl::flat_hash_map<std::string, std::vector<uint64_t>> function_hashes_to_hook_map,
       absl::flat_hash_map<std::string, std::vector<uint64_t>> frame_track_function_hashes_map);
-  orbit_base::Future<ErrorMessageOr<void>> LoadModule(const ModuleData* module);
-  orbit_base::Future<ErrorMessageOr<void>> LoadModule(const std::string& module_path,
-                                                      const std::string& build_id);
+
+  // LoadModule retrieves a module file and returns the local file path (potentially from the local
+  // cache). Symbols won't be loaded.
+  orbit_base::Future<ErrorMessageOr<std::filesystem::path>> LoadModule(const ModuleData* module);
+  orbit_base::Future<ErrorMessageOr<std::filesystem::path>> LoadModule(
+      const std::string& module_path, const std::string& build_id);
```

Comment:
The comments certainly clarify it, but from the name of the methods the word "load" might be bit overloaded. What do you think of `RetrieveModule` here and `RetrieveModuleAndLoadSymbols` below?

Reply:
Yes, that's a very good point. I also thought about this already and I like "Retrieve". Changed.